schema/me/notifications.js

```diff
@@ -46,6 +47,10 @@ const NotificationsFeedItemType = new GraphQLObjectType({
         return gravity('artworks', { ids: object_ids });
       },
     },
+    image: {
```

Comment:
I think what I need is the artist's image for the avatar, so maybe that would be `object.artist.image_url`?